### ISSUE TITLE
Add gettext to docs/installation/local.md

### ISSUE THREAD

**Issue by chase-west** [fossasia/open-event-frontend#9170]
**Is your feature request related to a problem? Please describe.**
It was difficult for me to find that gettext was needed to be manually downloaded on windows.

**Describe the solution you'd like**
Add info to get gettext in the local installation instructions. 

**Describe alternatives you've considered**
Make it more obvious in terminal.

**Comment by Blaize15** [comment]
Did you encounter `gettext` error while running /scripts/l10n.js script on your Windows machine? The instruction is already there for mac user.
<img width="660" alt="image" src="https://github.com/fossasia/open-event-frontend/assets/59837486/12e53d41-7607-4796-9e17-e4bc38bdb47b">

**Comment by chase-west** [comment]
Yes it was on windows. I'm aware the instruction is available for mac but I couldn't find anything on this repo about the windows installation for gettext.

**Comment by Blaize15** [comment]
> Yes it was on windows. I'm aware the instruction is available for mac but I couldn't find anything on this repo about the windows installation for gettext.

Got it! can you share the gettext error so that I can have better clarity to fix that and add that solution to the docs.

**Comment by chase-west** [comment]
![image](https://github.com/fossasia/open-event-frontend/assets/57733840/2b7fc05c-9070-4e26-91df-14319bd9c3bb)
This is displayed in the terminal after it is found that gettext is installed and asks to automatically install it. Unfortunately, this does not work on windows and that is only mentioned much higher in the command line here 
![image](https://github.com/fossasia/open-event-frontend/assets/57733840/893c4dea-07ce-4333-8eea-56fc655b204d) For someone like me who doesn't have a large terminal size it wasn't obvious that i was supposed to scroll and find this. Therefore, I think that this should be put in the actual documentation similarly to the mac documentation. I could do this if you'd like.
